astacus/coordinator/cleanup.py

```diff
@@ -22,10 +22,21 @@ async def create(*, c: Coordinator = Depends(), req: ipc.CleanupRequest = ipc.Cl
 
     def __init__(self, *, c: Coordinator, req: ipc.CleanupRequest) -> None:
         context = c.get_operation_context()
-        retention = c.config.retention.copy()
-        if req.retention is not None:
-            # This returns only non-defaults -> non-Nones
-            for k, v in req.retention.dict().items():
-                setattr(retention, k, v)
+        if req.retention is None:
+            retention = ipc.Retention(
+                minimum_backups=c.config.retention.minimum_backups,
```

Comment:
The config is still using pydantic, to avoid reaching up to the rohmu pydantic models that are external to astacus).
It's unpleasant to have two Retention types, but less than duplicating all the rohmu models (and keeping them in sync with the external definition).